app/src/main/java/com/example/travellens/fragments/ComposeFragment.java

```diff
@@ -201,7 +202,7 @@ public void onClick(View v) {
                     Exception exception = task.getException();
                     if (exception instanceof ApiException) {
                         ApiException apiException = (ApiException) exception;
-                        Log.e("populateChipsWithLocation", "Place not found: " + apiException);
+                        Log.e(TAG, "Place not found: " + apiException);
```

Comment:
Same here. Doing "+ apiException" will only log the message. Not the full stacktrace.